packages/codebytes/src/editor.tsx

```diff
@@ -22,9 +23,9 @@ const Output = styled.pre<{ hasError: boolean }>`
   font-family: Monaco;
   font-size: 0.875rem;
   overflow: auto;
-  ${({ hasError }) => `
+  ${({ hasError, theme }) => `
   color: ${hasError ? theme.colors.orange : theme.colors.white};
-  background-color: ${theme.colors['gray-900']};
+  background-color: ${colors['gray-900']};
```

Comment:
i forget which we prefer theme.colors or importing colors directly but we should go with one instead of both :)

Reply:
Yeah, this is a really interesting one! The `theme.colors['gray-900']`, is different color than the `${colors['gray-900']}`. We need the color that comes up in `${colors['gray-900']}` but, yeah, colors is definitely deprecated. I'll shoot a message to @dreamwasp and see what the best approach is for a deprecated color?

Reply:
oh thats not great. definitely should loop in gamut team for this

Reply:
spoke to cass, the new color is `${theme.colors['navy-900']}`, so we're covered!